examples/svm/plot_svm_scale_c.py

```diff
@@ -67,13 +57,25 @@
 )
 
 # %%
-# Now, we can define a linear SVC with the `l1` penalty.
+# L1-penalty case
+# ---------------
+# In the L1 case, theory says that prediction consistency (i.e. that under given
```

Comment:
Since we mention "theory says", I think we should refer to the article cited by Alex.

Reply:
I am not sure the claim "theory says" is justified in the cited documents, mostly because "prediction consistency" and "model consistency" aren't standard terms in the machine learning . I do cite the references in current lines 145 to 148, as they are more relevant at that level of the discussion.

I could still try to rephrase this paragraph to avoid such concepts and keep the underlying idea: L1 may set some coefficients to zero, reducing variance/increasing bias even in the limit where the sample size grows to infinity.

Reply:
Indeed, this would be nice.